hopsworks-common/src/main/java/io/hops/hopsworks/common/jobs/spark/SparkYarnRunnerBuilder.java

```diff
@@ -165,7 +165,7 @@ public YarnRunner getYarnRunner(Project project,
     );
 
     //Set executor extraJavaOptions to make parameters available to executors
-    Map<String, String> extraJavaOptions = new HashMap<>();
+    HashMap<String, String> extraJavaOptions = new HashMap<>();
```

Comment:
@robzor92 same as previous comment

Reply:
I added a HashMap argument containing jobs/jupyter specific jvm options to the setFrameworkProperties arguments. This former Map is passed in there. We anyway instantiate it as a HashMap.